### ISSUE TITLE
infinity loop if raid56 scrub is canceled

### ISSUE THREAD

**Issue by barthaare** [kdave/btrfsmaintenance#124]
I think raid56 scrub is waiting for scrub status `finished` to operate the next device. In case the scrub is canceled the status `finished` will not appear.  
`until btrfs scrub status "$DEV" | grep finished` line 52 in btrfs-scrub.sh

**Comment by kdave** [comment]
Right, the checks should be for more than just `finished`, effectively all results but `running`.

**Comment by kdave** [comment]
Fixed in devel, thanks.
